conformance/tests/httproute-rewrite-path.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: rewrite
```

Comment:
Recommend using a unique resource name here so we don't run into race conditions in between tests where a route of the same name is being deleting around the same time this is being created.